pom.xml

```diff
@@ -40,46 +65,58 @@
                     <groupId>com.fasterxml.jackson.core</groupId>
                     <artifactId>jackson-databind</artifactId>
                 </exclusion>
+
             </exclusions>
         </dependency>
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
-            <version>2.16.1</version>
+            <version>${jackson-databind.version}</version>
         </dependency>
         <dependency>
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-compress</artifactId>
-            <version>1.26.2</version>
+            <version>${commons-compress.version}</version>
```

Comment:
This is a temporary exclusion, isn't it?
You should add a comment next to this stating that it is being excluded to address the CVE until commons-compress is updated to pull in a version of commons-lang3 that does not show that CVE (version >= 3.18.0, right?)